packages/edit-post/src/components/secondary-sidebar/inserter-sidebar.js

```diff
@@ -44,6 +46,12 @@ export default function InserterSidebar() {
 					onClick={ () => setIsInserterOpened( false ) }
 				/>
 			</div>
+			<VisuallyHidden as="div">
+				<Button
+					label={ __( 'Close block inserter' ) }
+					onClick={ () => setIsInserterOpened( false ) }
+				/>
+			</VisuallyHidden>
```

Comment:
There's already a button right above it to close the inserter, can't we instead use aria-label there and avoid the duplicate element?

Reply:
@youknowriad Afraid not. At least not without some styling work and I doubt anyone wanted to take that on.

https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/components/secondary-sidebar/style.scss

As you can see, the CSS matches the class on the wrapping div and hides it. A better solution here would be when it is hidden visually, still display it to screen readers but not sure if it's possible or worth the work.

Reply:
@youknowriad You may like this approach better.

- Removed the CSS.
- The visual button will show and is now accessible to screen readers.
- The visually hidden kicks in when below medium viewport.

This seems to work just as well and now we don't have duplicate code.